### PR TITLE
LPS-88121 Set target platform 7.0 version to latest artifact syntax

### PR DESCRIPTION
**Pull request by codyhoag**
https://issues.liferay.com/browse/LPS-88121

### CODE REVIEW

File: modules/sdk/project-templates/project-templates-workspace/src/main/resources/archetype-resources/gradle.properties [hunk 1]
```diff
@@ -120,7 +120,7 @@
     # Liferay CE release and "7.1.10" for the latest DXP release.
     #
 #if (${liferayVersion.startsWith("7.0")})
-    #liferay.workspace.target.platform.version=7.0-GA7
+    #liferay.workspace.target.platform.version=7.0.6.1
```
Comment (gamerson) [inline modules/sdk/project-templates/project-templates-workspace/src/main/resources/archetype-resources/gradle.properties]: Lets do `7.0.6` and also lets update the documentation in `gradle.properties`

Comment (codyhoag) [reply modules/sdk/project-templates/project-templates-workspace/src/main/resources/archetype-resources/gradle.properties]: @gamerson two questions here:
1. According to Nexus, the latest version of the BOM artifact is `7.0.6.1`. I was of the understanding that this updated version fixed an issue with `7.0.6`. Am I misunderstanding how this works?
2. Do you want the documentation for this property to include both 7.0 and 7.1 **latest** CE and DXP released BOM versions? Is that what you mean by updating the documentation in `gradle.properties`?

Comment (gamerson) [reply modules/sdk/project-templates/project-templates-workspace/src/main/resources/archetype-resources/gradle.properties]: We fixed the 7.0.6 artifact on our servers, so it should be safe to use 7.0.6.

And yes we need to update the embedded documentation to just use the simpler version numbers.

**Comment by codyhoag** [comment]
@gamerson I've updated the version and descriptions. I just referred to version `7.0.10.7` as the latest DXP BOM for Liferay 7.0 (this is what was shown on Nexus). I know things are in flux with the SP and FP versions, so I stuck with the version-only syntax. Let me know if that should be modified.

**Comment by gamerson** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by gamerson** [comment]
Pull request submitted to https://github.com/gamerson/liferay-portal/pull/253. <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Pull request closed without merging**